src/Cohortmanager.js

```diff
@@ -0,0 +1,28 @@
+class Cohortmanager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  create(cohortName) {
```

Comment:
I would probably add guard clauses to your methods to throw an error when an input it invalid, for example, so you can guarantee you are getting the expected output and then test for this.